apps/src/aiComponentLibrary/userMessageEditor/UserMessageEditor.tsx

```diff
@@ -19,64 +19,75 @@ export interface UserMessageEditorProps {
   customPlaceholder?: string;
 }
 
-const UserMessageEditor: React.FunctionComponent<UserMessageEditorProps> = ({
-  onSubmit,
-  disabled,
-  editorContainerClassName,
-  customPlaceholder,
-  showSubmitLabel = false,
-}) => {
-  const [userMessage, setUserMessage] = useState<string>('');
-
-  const userMessageIsEmpty = useMemo(() => {
-    return userMessage.trim() === '';
-  }, [userMessage]);
+const UserMessageEditor = React.forwardRef<
+  HTMLDivElement,
+  UserMessageEditorProps
+>(
+  (
+    {
+      onSubmit,
+      disabled,
+      editorContainerClassName,
+      customPlaceholder,
+      showSubmitLabel = false,
+    },
+    ref
+  ) => {
+    const [userMessage, setUserMessage] = useState<string>('');
 
-  const handleKeyPress = (e: React.KeyboardEvent, userMessage: string) => {
-    if (e.key === 'Enter' && userMessage.trim() !== '') {
-      e.preventDefault(); // Prevent the text box from having just a blank line.
-      handleSubmit(userMessage);
-    }
-  };
+    const userMessageIsEmpty = useMemo(() => {
+      return userMessage.trim() === '';
+    }, [userMessage]);
 
-  const handleSubmit = useCallback(
-    (userMessage: string) => {
-      onSubmit(userMessage);
-      setUserMessage('');
-    },
-    [onSubmit]
-  );
+    const handleKeyPress = (e: React.KeyboardEvent, userMessage: string) => {
+      if (e.key === 'Enter' && userMessage.trim() !== '') {
+        e.preventDefault(); // Prevent the text box from having just a blank line.
+        handleSubmit(userMessage);
+      }
+    };
 
-  const icon = {iconName: 'paper-plane'};
-  return (
-    <div
-      className={classnames(
-        moduleStyles.editorContainer,
-        editorContainerClassName
-      )}
-    >
-      <textarea
-        id="uitest-chat-textarea"
-        className={moduleStyles.textArea}
-        placeholder={customPlaceholder || commonI18n.aiUserMessagePlaceholder()}
-        onChange={e => setUserMessage(e.target.value)}
-        value={userMessage}
-        disabled={disabled}
-        onKeyDown={e => handleKeyPress(e, userMessage)}
-      />
+    const handleSubmit = useCallback(
+      (userMessage: string) => {
+        onSubmit(userMessage);
+        setUserMessage('');
+      },
+      [onSubmit]
+    );
 
-      <div className={moduleStyles.centerSingleItemContainer}>
-        <Button
-          id="uitest-chat-submit"
-          isIconOnly={!showSubmitLabel}
-          onClick={() => handleSubmit(userMessage)}
-          disabled={disabled || !userMessage || userMessageIsEmpty}
-          text={showSubmitLabel ? commonI18n.submit() : undefined}
-          {...{[showSubmitLabel ? 'iconLeft' : 'icon']: icon}}
+    const icon = {iconName: 'paper-plane'};
+    return (
+      <div
+        ref={ref}
```

Comment:
Should the `ref` go on the `textarea` since that's what we want to focus? 

Edit: we also wouldn't need the `querySelector('textarea')` lookup then

Reply:
Updated!